_pages/introduction.md

```diff
@@ -0,0 +1,124 @@
+
+### Notice of Proposed Rulemaking (NPR) or Proposed Rule
+
+This is the initial draft of a change to a regulation. The notice starts with a “Preamble” which can contain the reasons for the regulation change, some section-by-section analysis, a cost analysis, and other general explanations. The Preamble is followed by the changes to the regulation. This notice only shows the actual changed parts of the regulation, not the new full text of the regulation.
+
+### Final Rule
+
+This is the final draft of a change to a regulation. This notice also starts with a Preamble and is followed by the changed parts of the regulation (not in context of the full text).
+
+### Interim Final Rule
+
+This is the same as a final rule except that it does not have to follow a proposed rule. This is uncommon. An example of this kind of rule: when CFPB was created, it reissued all of the regulations it was given authority over in full as interim final rules. There was no need for a notice and comment period.
+
+## <a name="structure"></a>The structure of a regulation
+
+Each regulation is part of the careful hierarchical structure of the [Code of Federal Regulations](https://www.gpo.gov/fdsys/browse/collectionCfr.action?collectionCode=CFR), and eRegulations reflects this structure. Here's a guide from the top down.
+
+### The Code of Federal Regulations contains “titles”
+
+The Code of Federal Regulations is broken into numeric “titles”, which each describe one subject area (such as “Domestic Security” or “Banks and Banking”). Agencies will most likely only cover one title. (1-50 is usually written in Roman numerals.)
+
+### Each title contains “parts” (“regulations”)
+
+CFR titles are broken into “parts”, which eRegulations consistently defines as the breaking point for a “regulation” (in other words: a “regulation” is a CFR part, for our purposes). These are numbered and further specify area of focus. A single agency will most likely cover multiple CFR parts. Technically, there can be a layer of categorization in between CFR titles and parts, but we ignore that for now. The format for this is `XX CFR XXX` or `title-number CFR part-number`.
+
+### Each part (regulation) contains components including “sections”
+
+A CFR part (or “regulation”) is further broken down into (often) Subparts, Subject Groups, Appendices, and Interpretations. Each of these components is optional, with Subparts and Appendices being lettered, and Interpretations technically being “Supplement I” within the Appendix.
+
+Subparts and Subject Groups are further broken into “sections”. Regulations without explicit subparts or subject groups will still have “sections.” These can be considered chapters in the book that is the regulation. The format for this is `XX CFR XXX.XsectionX`.
+
+### Each section contains paragraphs and subparagraphs
```

Comment:
We should probably say "Many sections contain paragraphs and subparagraphs". Many ATF sections, for example, are a single block of text.